angular/core/components/thread_page/thread_item/thread_item.coffee

```diff
@@ -1,10 +1,23 @@
-angular.module('loomioApp').directive 'threadItem', ($translate, LmoUrlService, EventHeadlineService) ->
-  scope: {event: '='}
+angular.module('loomioApp').directive 'threadItem', ($compile, $translate, LmoUrlService, EventHeadlineService) ->
+  scope: {event: '=', threadWindow: '=?', root: '='}
   restrict: 'E'
   templateUrl: 'generated/components/thread_page/thread_item/thread_item.html'
+
+  link: (scope, element, attrs) ->
+    if scope.root
+      $compile("<event-children parent=\"event\" thread-window=\"threadWindow\"></event-children>")(scope, (cloned, scope) -> element.append(cloned))
```

Comment:
what will kill the perf? it does not seem to do so for me, or be talked about online.. 

we need it because threadItem refers to threadItem (via eventchildren) which angular cannot handle without this.